### PR TITLE
Upstream deploy chef fix

### PR DESCRIPTION
**Pull request by haad**

### CODE REVIEW

File: littlechef/runner.py [hunk 1]
```diff
@@ -250,6 +228,8 @@ def recipe(recipe):
     data = lib.get_node(env.host_string)
     data["run_list"] = ["recipe[{0}]".format(recipe)]
     if not __testing__:
+        if env.autodeploy_chef and not chef.chef_test():
+                deploy_chef(ask="no")
```
Comment (haad) [inline littlechef/runner.py]: Why ? __testing__ if was there before and we want to install chef client only when we have autodeploy set and it's not installed yet.

Comment (tobami) [inline littlechef/runner.py]: with "twice" I meant 8 spaces instead of 4

**Comment by tobami** [comment]
Great, apart from the 8 spaces indentation of line 232 on runner.py

**Comment by haad** [comment]
Ok I finally got time to fix this one too yay!

**Comment by tobami** [comment]
:sparkles: Great work!

**Pull request merged**